packages/table/src/Cell/Cell.styles.ts

```diff
@@ -75,35 +74,43 @@ export const basicCellStyles = css`
   }
 `;
 
-export const cellContentContainerStyles = css`
+export const cellTransitionContainerStyles = css`
   display: flex;
   align-items: center;
-  text-overflow: ellipsis;
   min-height: ${standardCellHeight}px;
-  transition-property: min-height, max-height, opacity, transform, padding;
+  transition-property: min-height, max-height, opacity, padding, transform;
   transition-duration: ${transitionDuration.default}ms;
   transition-timing-function: ease;
 `;
 
-export const disableAnimationStyles = css`
-  transition: none;
+export const truncatedContentStyles = css`
+  /* See https://css-tricks.com/line-clampin/#aa-the-standardized-way */
+  display: -webkit-box;
+  -webkit-line-clamp: ${standardCellHeight / typeScales.body1.lineHeight};
+  -webkit-box-orient: vertical;
+  -webkit-box-align: start;
 `;
 
-const _hiddenStyles = css`
-  opacity: 0;
-  min-height: 0;
-  max-height: 0;
-  overflow: hidden;
+export const disableAnimationStyles = css`
+  transition-duration: 0;
+  transition: none;
 `;
 
-export const cellContentTransitionStyles = (
-  height: number,
+export const cellContentTransitionStateStyles = (
+  height?: number,
 ): Record<TransitionStatus, string> => {
+  const _hiddenStyles = css`
+    opacity: 0;
+    min-height: 0;
+    max-height: 0;
+    overflow: hidden;
+  `;
+
   return {
     entered: css`
       opacity: 1;
       min-height: ${standardCellHeight}px;
-      max-height: ${height}px;
+      max-height: ${height + 'px' ?? 'unset'};
```

Comment:
I think you mean `height ? `${height}px : 'unset'}`

Reply:
Im having trouble formatting that properly sorry

Reply:
Neither of us had it the first time 😆  

`height ? height + "px" : 'unset'`  is what I was going for